.readthedocs.yml

```diff
@@ -0,0 +1,14 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/source/conf.py
+  fail_on_warning: false
```

Comment:
>I think false is the default value, so if ADS is going with that choice, then this line might not be needed.

I just copied SCT's, which had it to true, so I just set it to false not knowing I could delete it in the first place. I think it's still nice to have, so devs see that as an option.


>Still though, I'm curious: Why false? At SCT we have this on true to catch issues in the docs. (Without it, we found that PRs would subtly break the docs, yet we would only catch the issues long after the offending PR was merged.)

I set it to false because I noticed we currently have outstanding warnings thrown during the build, and was prioritizing getting RTD building again ASAP over handling some bad/legacy RTD syntax issues in this branch. I could open another issue to set it to true and handle those issues separately.

Reply:
> I set it to false because I noticed we currently have outstanding warnings thrown during the build, and was prioritizing getting RTD building again ASAP over handling some bad/legacy RTD syntax issues in this branch.

Totally fair!

> I could open another issue to set it to true and handle those issues separately.

+1 to this. :)

Reply:
Opened #579 to handle this.